docs_src/microservices/device/supported/device-onvif-camera/supplementary-info/ONVIF-protocol.md

```diff
@@ -14,9 +15,28 @@ This Onvif Camera Device Service is developed to control/communicate ONVIF-compl
 The latest version {{version}} of the device service API specifications can be found
 [here](../swagger.md).
 
-
-
-## Onvif Features
+## ONVIF Protocol Properties
+
+| Property | Description | EdgeX Value Type |               
+| -- | -- | -- |  
+| Address | The IP address of the camera. | String |  
+| Port | The internet port of the camera. | String |  
+| EndpointRefAddress | A GUID used to indentify the camera endpoint. | String |  
+| DeviceStatus | The discovery/authorization status of the camera.  | String |  
```

Comment:
I think `DeviceStatus` section can be changed.  This description is better https://docs.edgexfoundry.org/3.0/microservices/device/supported/device-onvif-camera/supplementary-info/device-status/